flutter/lib/src/event_processor/platform_exception_event_processor.dart

```diff
@@ -0,0 +1,115 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+import 'package:sentry/sentry.dart';
+import 'package:stack_trace_parser/stack_trace_parser.dart';
+
+class AndroidPlatformExceptionEventProcessor implements EventProcessor {
+  const AndroidPlatformExceptionEventProcessor();
+
+  @override
+  FutureOr<SentryEvent?> apply(SentryEvent event, {hint}) async {
+    final plaformException = event.throwable;
+    if (!(plaformException is PlatformException)) {
+      return event;
+    }
+
+    final nativeStackTrace = plaformException.stacktrace;
+    if (nativeStackTrace == null) {
+      return event;
+    }
+
+    // PackageInfo has an internal cache, so no need to do it ourselves.
+    final packageInfo = await PackageInfo.fromPlatform();
+    return _processPlatformException(
+      event,
+      plaformException,
+      nativeStackTrace,
+      packageInfo.packageName,
+    );
+  }
+
+  SentryEvent _processPlatformException(
+    SentryEvent event,
+    PlatformException exception,
+    String nativeStackTrace,
+    String packageName,
+  ) {
+    final e =
+        _JvmExceptionFactory(packageName).fromJvmStackTrace(nativeStackTrace);
+
+    return event.copyWith(
+      exceptions: [
+        ...?event.exceptions,
+        ...e,
+      ],
+    );
+  }
+}
+
+class _JvmExceptionFactory {
+  const _JvmExceptionFactory(this.nativePackageName);
+
+  final String nativePackageName;
+
+  List<SentryException> fromJvmStackTrace(String exceptionAsString) {
+    final jvmException = JvmException.parse(exceptionAsString);
+    final jvmExceptions = <JvmException>[
+      jvmException,
+      ...?jvmException.causes,
+      ...?jvmException.suppressed,
+    ];
+
+    return jvmExceptions.map((exception) {
+      return exception.toSentryException(nativePackageName);
+    }).toList(growable: false);
+  }
+}
+
+extension on JvmException {
+  SentryException toSentryException(String nativePackageName) {
+    return SentryException(
```

Comment:
missing the `module`, which is the package of the class, e.g. `java.lang.IllegalArgumentException`, module is `java.lang`.